DPGAnalysis/HcalTools/scripts/cmt/condor.jdl

```diff
@@ -8,7 +8,7 @@ arguments  = runlist_run $(ClusterID) $(ProcId) $(Proxy_path)
 Should_Transfer_Files = YES
 WhenToTransferOutput = ON_EXIT
 
-Transfer_Input_Files = run_condor.csh,mkcfg_new120.csh,file_lists.csh,a.py.beg,a.py.end,runlist_run,/afs/cern.ch/work/z/zhokin/hcal/voc2/CMSSW_11_1_0_pre3/src/DPGAnalysis/HcalTools/src/CMTRawAnalyzer.cc,/afs/cern.ch/work/z/zhokin/hcal/voc2/CMSSW_11_1_0_pre3/src/DPGAnalysis/HcalTools/interface/CMTRawAnalyzer.h,/afs/cern.ch/work/z/zhokin/hcal/voc2/CMSSW_11_1_0_pre3/src/DPGAnalysis/HcalTools/BuildFile.xml
+Transfer_Input_Files = run_condor.csh,mkcfg_new120.csh,file_lists.csh,a.py.beg,a.py.end,runlist_run,/afs/cern.ch/work/z/zhokin/hcal/qq6/CMSSW_12_2_0_pre1/src/DPGAnalysis/HcalTools/src/CMTRawAnalyzer.cc,/afs/cern.ch/work/z/zhokin/hcal/qq6/CMSSW_12_2_0_pre1/src/DPGAnalysis/HcalTools/interface/CMTRawAnalyzer.h,/afs/cern.ch/work/z/zhokin/hcal/qq6/CMSSW_12_2_0_pre1/src/DPGAnalysis/HcalTools/BuildFile.xml
```

Comment:
CMTRawAnalyzer.cc is now in `\plugin,` not in `\src` (just pointing it out here, because I don't fully understand what this line is meant to...)

Reply:
Thanks for the valuable comment. But in any case, all codes required for running will still be adjusted according to the instructions in Readme.